libraries/src/Updater/Adapter/ExtensionAdapter.php

```diff
@@ -183,7 +183,7 @@ protected function _endElement($parser, $name)
 								'JLIB_INSTALLER_AVAILABLE_UPDATE_DB_TYPE',
 								$this->currentUpdate->name,
 								$this->currentUpdate->version,
-								Text::_($db->name)
+								Text::_('JLIB_DB_SERVER_TYPE_' . $dbType)
```

Comment:
```suggestion
								Text::_('JLIB_DB_SERVER_TYPE_' . $dbTypeUcase)
```
Can be done if my previous suggestion gets accepted.